tests/binary/script.R

```diff
@@ -0,0 +1,25 @@
+# Loaded with --default-packages=
```

Comment:
I think you want the list to be explicitly overridden in the BUILD file for the test rather than rely on the default, which should be able to change without breaking the test.